sunpy/instr/aia/response.py

```diff
@@ -0,0 +1,279 @@
+    """
+    This definition reads the .genx output file from SolarSoft Ware (SSW). It will extract the instrument data and save them to an data file to be used for this program.
+
+     Parameters
+    ----------
+    :param path_to_file: string, the path location that leads to ssw_aia_response_genx file documents. The output file
+                    used for obtaining all of the information is:   aia_V6_all_fullinst
+    :param outfile_name: string, the name of the outfile that will store the dataframe
+                                    - could make this a dictioary called self.instr_properties in class format
+
+    Returns
+    -------
+    output: a comma separated dataframe file with arrays for each property per channel
+
+    """
+
+    # access outer dictionary from .genx file
+    ssw_dict = readsav(path_to_file)
+
+    # access inner dictionary from .genx file
+    data = ssw_dict['data']
+
+    # define wanted properties
+    indexes = ['wave', 'effarea', 'units', 'geoarea', 'scale', 'platescale', 'numfilters',  'wavemin', 'wavestep', 'wavenumsteps', 'wavelog', 'filtersincludemesh', 'meshtrans', 'usecontam', 'contamthick',  'usephottoelec','elecperev','usephottodn', 'elecperdn', 'useerror', 'fp_filter', 'ent_filter', 'primary', 'secondary', 'ccd', 'contam', 'cross_area']
+    channels = ['A94', 'A131', 'A171', 'A193', 'A211', 'A304', 'A335']
+
+    # write into dataframe for each channel from inner dictionaries
+    df = pd.DataFrame(index=indexes, columns=channels)
+
+    for channel in channels:
+
+        # pick out channel files to look into
+        for name in data.dtype.names:
+            if name.startswith('A') and name.endswith('THICK_FULL') and channel in name:
+                key = channel
+
+                # store information from those files in dataframe
+                for value in data[name]:
+                    for prop in indexes:
+                        df.loc[prop] = [value[prop][0]]
+
+    # save to dataframe outfile
+    df.to_csv('channel_properties.csv', sep=',')
+
+
+
+def load_channel_as_dict(channel = 'all'):
+    """
+    This definition reads the .csv output file from the read_genx function. It will return a dictionary for each channel
+
+     Parameters
+    ----------
+    :param channel, string or int, the specific channel to get properties of from the .csv file
+
+    Returns
+    -------
+    output: a dictionary of one channel is returned where it has keys of the properties of the channel
+    """
+
+    dataframe = pd.DataFrame.from_csv('channel_properties.csv', sep = ',')
+
+    # load in data for one channel and save into a dictionary
+    one_channel_dict = {}
+    for key,value in dataframe.iteritems():
+        if type(channel) == int:
+            channel = "A" + str(channel)
+        if channel == str(key):
+            one_channel_dict[key] = value
+
+    assert len(one_channel_dict) != 0
+    return one_channel_dict
+
+
+
+
+def effective_area(channel = 'A131'):
```

Comment:
is it channel a wavelength? should we use units here? I can see where this wouldn't be... but I am just want to be sure we are all with the same idea.